### ISSUE TITLE
Publish source files with `.js.flow` extensions

### ISSUE THREAD

**Issue by bassettsj** [react-grid-layout/react-draggable#184]
Great library! Since you're already using flow type on the source files it would be really great if you published the source files with the `.js.flow` extensions for the matching main files in the directories. It would just be a simple change in the make file to allow this. Any consumers that are also using flow would get access to the flow types that you have already added 🎉 !

Would you be interested in a PR to add this?

**Comment by STRML** [comment]
Hey @bassettsj - we're already using [transform-flow-comments](https://github.com/mzabriskie/react-draggable/blob/master/.babelrc#L8), so Flow should already be picking this up on its own.

**Comment by bassettsj** [comment]
@STRML It should but flow is unable to understand the webpack wrapper that is used in the build process. I get no type info on its own. 😦

**Comment by STRML** [comment]
Ah. Perhaps this is a good reason to move to Rollup.

**Comment by bassettsj** [comment]
Yeah I suppose it might work. Even with a simple babel cli transform it wasn't able to pickup the types as expected. Hmm... I think the `.js.flow` might be the best way for flow to pick up the right types.

**Comment by STRML** [comment]
Also - what are you trying to typecheck? Just the props? I've never seen Flow actually be able to pull propTypes out of an imported component.

**Comment by STRML** [comment]
The [rollup branch](https://github.com/mzabriskie/react-draggable/tree/rollup) is almost there but Rollup is pretty broken. Making a tracking issue.

**Comment by bassettsj** [comment]
@STRML Sorry about the delay. I added a simple [sandbox project](https://github.com/bassettsj/example-project) and repository that shows that `.js.flow` style of importing types works pretty well. The results are as expected that I should be told that I need to add "children" and "title" to the element.

``` js

index.js:7
  7: render(<Sandbox />, document.querySelector('#app'));
            ^^^^^^^^^^^ React element `Sandbox`
  6:     props: {
                ^ property `children`. Property not found in. See: node_modules/sandbox-flow-js/index.js.flow:6
  7: render(<Sandbox />, document.querySelector('#app'));
            ^^^^^^^^^^^ props of React element `Sandbox`

index.js:7
  7: render(<Sandbox />, document.querySelector('#app'));
            ^^^^^^^^^^^ React element `Sandbox`
  6:     props: {
                ^ property `title`. Property not found in. See: node_modules/sandbox-flow-js/index.js.flow:6
  7: render(<Sandbox />, document.querySelector('#app'));
            ^^^^^^^^^^^ props of React element `Sandbox`

```

here is the really simple module it is consuming https://github.com/bassettsj/sandbox-flow-js
